pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -133,7 +134,7 @@ def list_variable(self):
         """
         return self._list_variable
 
-#==============================================================================
+#=======================================================================================
```

Comment:
Same comment

Reply:
fixed in 12cd0b2a08f4f7d59e7c61e94769a7a9d9092039